client/web/src/batches/RepoBatchChangesButton.story.tsx

```diff
@@ -15,19 +14,36 @@ const config: Meta = {
 }
 
 export default config
-
+let openValue = 0
+let mergedValue = 0
 const queryRepoChangesetsStats: typeof _queryRepoChangesetsStats = () =>
     of({
         changesetsStats: {
-            open: number('Open', 2),
-            merged: number('Merged', 47),
+            open: openValue,
+            merged: mergedValue,
         },
     })
 
-export const RepoButton: Story = () => (
+export const RepoButton: Story = args => (
     <WebStory>
-        {() => <RepoBatchChangesButton repoName="Awesome Repo" queryRepoChangesetsStats={queryRepoChangesetsStats} />}
+        {() => (
+            <>
+                {(openValue = args.open)}
+                {(mergedValue = args.merged)}
+                <RepoBatchChangesButton repoName="Awesome Repo" queryRepoChangesetsStats={queryRepoChangesetsStats} />
+            </>
+        )}
```

Comment:
```suggestion
        {() => {
            openValue = args.open
            mergedValue = args.merged
            
            return <RepoBatchChangesButton repoName="Awesome Repo" queryRepoChangesetsStats={queryRepoChangesetsStats} />
        }}
```

These assignments will need to be done outside of the JSX that's rendered. Right now these numbers also get rendered as plain text to the left of the component:

<img width="340" alt="image" src="https://user-images.githubusercontent.com/10523985/182913321-a231c32c-9a25-4476-8a6b-d48593a7436a.png">